[2022-08-06 19:38:42,279] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 19:38:42,294] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 19:38:42,295] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:42,295] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:42,295] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:42,311] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-20 00:00:00+00:00
[2022-08-06 19:38:42,317] {standard_task_runner.py:52} INFO - Started process 2600 to run task
[2022-08-06 19:38:42,319] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-20T00:00:00+00:00', '--job-id', '798', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpxin8mu5w', '--error-file', '/tmp/tmpbu17d6wr']
[2022-08-06 19:38:42,320] {standard_task_runner.py:80} INFO - Job 798: Subtask second_task
[2022-08-06 19:38:42,395] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:42,486] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-20T00:00:00+00:00
[2022-08-06 19:38:42,487] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:42,487] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:42,498] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:42,500] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:42,500] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:42,531] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220420T000000, start_date=20220806T193842, end_date=20220806T193842
[2022-08-06 19:38:42,571] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:42,615] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:33,105] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 21:52:33,121] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 21:52:33,121] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:33,121] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:33,121] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:33,139] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-20 00:00:00+00:00
[2022-08-06 21:52:33,146] {standard_task_runner.py:52} INFO - Started process 2782 to run task
[2022-08-06 21:52:33,149] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-20T00:00:00+00:00', '--job-id', '797', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp8rgdls__', '--error-file', '/tmp/tmp4y1kpgbu']
[2022-08-06 21:52:33,149] {standard_task_runner.py:80} INFO - Job 797: Subtask second_task
[2022-08-06 21:52:33,222] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:33,315] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-20T00:00:00+00:00
[2022-08-06 21:52:33,315] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:33,316] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:33,328] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:33,330] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:33,330] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:33,362] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220420T000000, start_date=20220806T215233, end_date=20220806T215233
[2022-08-06 21:52:33,400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:33,431] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:00,317] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 21:58:00,332] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 21:58:00,332] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:00,333] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:00,333] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:00,353] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-20 00:00:00+00:00
[2022-08-06 21:58:00,359] {standard_task_runner.py:52} INFO - Started process 2594 to run task
[2022-08-06 21:58:00,362] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-20T00:00:00+00:00', '--job-id', '803', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp9fzh7it5', '--error-file', '/tmp/tmpzb_lcgh5']
[2022-08-06 21:58:00,362] {standard_task_runner.py:80} INFO - Job 803: Subtask second_task
[2022-08-06 21:58:00,452] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:00,578] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-20T00:00:00+00:00
[2022-08-06 21:58:00,579] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:00,580] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:00,596] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:00,598] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:00,598] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:00,649] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220420T000000, start_date=20220806T215800, end_date=20220806T215800
[2022-08-06 21:58:00,694] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:00,738] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:01,785] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 22:13:01,804] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 22:13:01,805] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:01,805] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:01,805] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:01,825] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-20 00:00:00+00:00
[2022-08-06 22:13:01,833] {standard_task_runner.py:52} INFO - Started process 2600 to run task
[2022-08-06 22:13:01,836] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-20T00:00:00+00:00', '--job-id', '800', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpmjlqqmej', '--error-file', '/tmp/tmp1955wzeb']
[2022-08-06 22:13:01,836] {standard_task_runner.py:80} INFO - Job 800: Subtask second_task
[2022-08-06 22:13:01,909] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:01,994] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-20T00:00:00+00:00
[2022-08-06 22:13:01,995] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:01,996] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:02,010] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:02,012] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:02,012] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:02,051] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220420T000000, start_date=20220806T221301, end_date=20220806T221302
[2022-08-06 22:13:02,087] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:02,156] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:20,250] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 22:24:20,267] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-08-06 22:24:20,267] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:20,267] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:20,268] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:20,289] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-20 00:00:00+00:00
[2022-08-06 22:24:20,296] {standard_task_runner.py:52} INFO - Started process 2657 to run task
[2022-08-06 22:24:20,299] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-20T00:00:00+00:00', '--job-id', '804', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpq7t5nd4e', '--error-file', '/tmp/tmp4h5y6k5y']
[2022-08-06 22:24:20,299] {standard_task_runner.py:80} INFO - Job 804: Subtask second_task
[2022-08-06 22:24:20,367] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-20T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:20,449] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-20T00:00:00+00:00
[2022-08-06 22:24:20,450] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:20,451] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:20,465] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:20,466] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:20,466] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:20,501] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220420T000000, start_date=20220806T222420, end_date=20220806T222420
[2022-08-06 22:24:20,550] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:20,598] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
